### ISSUE TITLE
lint and test are missing in GH workflow

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/cml#522]
We have removed the ability to pass the tests and linter in dev branches to warranty the healthiness of the code we pretend to merge or accept PRs

```sh
npm run lint
npm run test
```

also checkbot has some inline comments that should be removed or investigate why they have been commented

```yaml
#cml-send-comment report.md
#cml-send-github-check report.md
```

**Comment by 0x2b3bfa0** [comment]
To be solved along with #183

**Comment by 0x2b3bfa0** [comment]
https://github.com/iterative/cml/blob/d716d3072d4d9829865d95f67bdf79cc7c9c5a62/.github/workflows/checkbot.yml#L59-L60

Uncommenting the `cml-send-comment` line doesn't cause any error, and the command produces the expected results. The `cml-send-github-check` line seems to produce the following error:

```console
$ REPO_TOKEN=${{secrets.TEST_GITHUB_TOKEN}} cml-send-github-check report.md
RequestError [HttpError]: You must authenticate via a GitHub App.
```

**Comment by 0x2b3bfa0** [comment]
Fixed with #539: it only works with the session-specific `GITHUB_TOKEN`
